File: Resources/InheritanceTAPApp/Customer.cs

using ECommerce;
namespace CRM;
public class Customer:Person{

    private string deliveryAddress;
    private string billingAddress;

    //Property
    public string DeliveryAddress{
        get { return this.deliveryAddress;}
        set{ this.deliveryAddress=value;}
    }

    public string BillingAddress{
        get { return this.billingAddress;}
        set{ this.billingAddress=value;}
    }


    public override string ToString(){
     string data=string.Format("{0}, {1} ",deliveryAddress,billingAddress);
     return base.ToString() + "  "+ data;
    }
}

File: Resources/Java/src/main/java/com/example/Customer.java
package com.example;

public class Customer extends Person {

    private String deliveryAddress;
    private String billingAddress;

    public String  getDeliveryAddress() { return this.deliveryAddress;}
    public void setDeliveryAddress(String address){ this.deliveryAddress=address;}

    public String  getBillingAddress() { return this.billingAddress;}
    public void  setBillingAddress(String address){ this.billingAddress=address;}

    @Override
    public  String toString()
    {
        String data= " " + this.deliveryAddress + " " + this.billingAddress;
        return data;
    }
}
